### ISSUE TITLE
Meteor 1.5 getstorybook error Block-scoped declarations (let, const, function, class) not yet supported outside strict mode

### ISSUE THREAD

**Issue by CJjumper** [storybooks/storybook#1466]
I keep running into this error with Meteor 1.5. 

```
/usr/local/lib/node_modules/@storybook/cli/bin/generate.js:42
let projectType;
^^^

SyntaxError: Block-scoped declarations (let, const, function, class) not yet supported outside strict mode
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:373:25)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Function.Module.runMain (module.js:441:10)
    at startup (node.js:139:18)
    at node.js:974:3
```

And yes I have read that all I have to do is upgrade node. But with with Meteor 1.5 am stuck at node version 4 for now. You guys could not just use "use strict" at the top of your files? ``

**Comment by dpankros** [comment]
+1  Also getting this

**Comment by plitex** [comment]
+1 Having same issue

**Comment by codeHatcher** [comment]
same issue

**Comment by ndelangen** [comment]
Would we only need to add 'use-strict' in the generator for meteor? or all over the place?

**Comment by Hypnosphi** [comment]
lack of use strict isn't the only problem, there's also some unsupported syntax like destructuring. So I decided that adding babel-register with env present is a better solution.

Plus it allows to move from commonjs to es imports

**Comment by Hypnosphi** [comment]
It's fixed in `3.3.0-alpha.2`
@shilman should I close?
